### PR TITLE
PUBDEV-5130: set the timezone for parsing datetime columns to be UTC …

### PR DESCRIPTION
**Pull request by rpeck**
…by default.

### CODE REVIEW

File: h2o-core/src/main/java/water/H2O.java [hunk 1]
```diff
@@ -2038,6 +2039,10 @@ public static void main( String[] args ) {
     if (GA != null)
       startGAStartupReport();
 
+    // Set parser timezone to UTC by default
+    ParseTime.setTimezone("UTC");
```
Comment (michalkurka) [inline h2o-core/src/main/java/water/H2O.java]: You can avoid send the MRTask simply by setting the private static property:
```
private static DateTimeZone _timezone = DateTimeZone.forID("URC")
```

File: h2o-py/h2o/h2o.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def connect(server=None, url=None, ip=None, port=None, https=None, verify_ssl_ce
                                      auth=auth, verify_ssl_certificates=verify_ssl_certificates,
                                      proxy=proxy,cookies=cookies,
                                      verbose=verbose)
-        h2oconn.cluster.timezone = "UTC"
+        # h2oconn.cluster.timezone = "UTC"  It's now set to UTC by default
```
Comment (michalkurka) [inline h2o-py/h2o/h2o.py]: I don't think the comment is necessary. Can we include the cluster time zone in the cluster status?

```
    H2O cluster uptime:         5 seconds 679 milliseconds 
    H2O cluster version:        3.17.0.99999 
    H2O cluster version age:    1 day  
    H2O cluster name:           mkurka 
    H2O cluster total nodes:    1 
    H2O cluster total memory:   3.56 GB 
    H2O cluster total cores:    8 
    H2O cluster allowed cores:  8 
    H2O cluster healthy:        TRUE 
    H2O Connection ip:          localhost 
    H2O Connection port:        54321 
    H2O Connection proxy:       NA 
    H2O Internal Security:      FALSE 
    H2O API Extensions:         AutoML, XGBoost, Algos, Core V3, Core V4 
    R Version:                  R version 3.3.1 (2016-06-21) 
```

File: h2o-core/src/main/java/water/H2O.java (hunk 1 above)
Comment (rpeck) [reply h2o-core/src/main/java/water/H2O.java]: The MRTask.doAllNodes() is necessary to make the update on all the nodes in the cluster, not just the master node.

File: h2o-py/h2o/h2o.py (hunk 2 above)
Comment (rpeck) [reply h2o-py/h2o/h2o.py]: I like the comment, to explain the change, but it's not necessary.

There are two timezones: one in DateTimeZone for the cluster itself and used for logging, and one for datafile parsing.  I suppose we could add those, but to me it seems overkill.

**Comment by rpeck** [comment]
@michalkurka: I removed the comment from the Python.

File: h2o-core/src/main/java/water/H2O.java (hunk 1 above)
Comment (michalkurka) [reply h2o-core/src/main/java/water/H2O.java]: It is not necessary :) All nodes have same copy of the code, all nodes will be statically initialized ;)

**Comment by michalkurka** [comment]
@rpeck thank you, please also read my other comment

File: h2o-core/src/main/java/water/H2O.java (hunk 1 above)
Comment (rpeck) [reply h2o-core/src/main/java/water/H2O.java]: Ah, good point.  I didn't look carefully enough to see that it was a static initializer.

**Comment by rpeck** [comment]
Simplified the initialization of _timezone.
R and Python now print the timezones from the server when they connect, as @michalkurka suggested above.

**Comment by rpeck** [comment]
Added doc ticket https://0xdata.atlassian.net/browse/PUBDEV-5149

**Pull request merged**